File: app/src/main/java/fr/erictruong/material/training/ui/core/fragment/DummyFragment.java
package fr.erictruong.material.training.ui.core.fragment;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import fr.erictruong.material.training.R;

public class DummyFragment extends Fragment {

    public static DummyFragment newInstance() {
        Bundle args = new Bundle();
        DummyFragment fragment = new DummyFragment();
        fragment.setArguments(args);
        return fragment;
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_dummy, container, false);
    }
}

File: app/src/main/java/fr/erictruong/material/training/ui/core/fragment/DummyFragment.kt
package fr.erictruong.material.training.ui.core.fragment

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.Gravity
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import org.jetbrains.anko.*
import org.jetbrains.anko.support.v4.nestedScrollView

class DummyFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return DummyFragmentUI().createView(AnkoContext.create(context, this))
    }
}

class DummyFragmentUI : AnkoComponent<Fragment> {

    override fun createView(ui: AnkoContext<Fragment>) = with(ui) {
        nestedScrollView {
            linearLayout {
//                gravity = Gravity.CENTER
//
//                textView("Dummy") {
//                    setAllCaps(true)
//                }
            }
        }
    }
}